source/unified-test-format/unified-test-format.rst

```diff
@@ -3909,6 +3910,10 @@ if desired:
 * ``errorCode``: Optional integer, which is unset by default. If set, the
   command will not be executed and the specified command error code will be
   returned as a command error.
+* ``errorLabels``: Optional array of strings. If set, this list overrides the
+  server's normal behavior for adding error labels. An empty array may be used
+  to suppress the server from adding error labels to the response. New in server
+  4.3.1 (`SERVER-43941 <https://jira.mongodb.org/browse/SERVER-43941>`__).
```

Comment:
Lifted from https://github.com/mongodb/mongo/wiki/The-%22failCommand%22-fail-point#failcommand-parameters